docs/jax.scipy.rst

```diff
@@ -106,6 +106,7 @@ jax.scipy.special
 .. autosummary::
   :toctree: _autosummary
 
+   betaln
```

Comment:
Move down one line for alphabetization

Reply:
WOW. It wasn't that early in the morning that I couldn't do my ABC's ... or so I thought. 🙏